plugins/inputs/opcua/README.md

```diff
@@ -187,6 +191,64 @@ to use them.
   #   # use_unregistered_reads = false
 ```
 
+## Client Certificate Configuration
+
+When using security modes other than "None", Telegraf acts as an OPC UA client
+and requires a client certificate to authenticate itself to the server. The
+plugin supports three certificate management approaches:
+
+### Temporary Self-Signed Certificates (Default)
+
+If both `certificate` and `private_key` options are left empty or commented
+out, Telegraf will automatically generate a self-signed certificate in a
+temporary directory on each startup:
+
+```toml
+# certificate = "/etc/telegraf/cert.pem"  # commented out
+# private_key = "/etc/telegraf/key.pem"   # commented out
+```
```

Comment:
This is a bit misleading because it will also behave this way if you delete the lines entirely...

```suggestion
temporary directory on each startup.
```